.github/workflows/ci-e2e.yaml

```diff
@@ -65,11 +65,14 @@ jobs:
         run: |
           export OCM_SINGLE=1
           make local-setup
-      - name: Deploy MGC
+      - name: Deploy
         run: |
-          make docker-build-controller kind-load-controller deploy-controller
+          make docker-build-policy-controller kind-load-policy-controller deploy-policy-controller
```

Comment:
I don't think you need `deploy-policy-controller` here since `deploy-gateway-controller` will deploy both anyway

Reply:
OK, i think for this to work with the current makefiles you need to do all the build and load tasks before you run any deploy one or you will end up with part of the deployment using the wrong thing.

```
make docker-build-gateway-controller kind-load-gateway-controller docker-build-policy-controller kind-load-policy-controller deploy-gateway-controller
```

Reply:
@maleck13 Can you add back the `separate-policy-controller` branch in this workflow so we can check it is actually working as expected.

Reply:
done